src/encoded/static/libs/formatMeasurement.js

```diff
@@ -0,0 +1,29 @@
+
+/**
+* Takes a magnitude and unit (singular form) and returns the measurement
+* with the proper unit in singular or plural, depending on magnitude.
+* Invalid measurements are turned as- Unknown
+* @param {string} magnitude - Measured value, range or unknown
+* @param {string} unit - Measurement system in singular form
+* @return {string} - Measurement with singular or plural unit
+* (depending on value/range) or "Unknown" without unit
+*/
+const formatMeasurement = (magnitude, unit) => {
+    if (!magnitude || (magnitude.toLowerCase && magnitude.toLowerCase()) === 'unknown') {
+        return 'Unknown';
+    }
+
+    // parseFloat used because parseInt only keeps integer
+    // remember, in JavaScript, 1 === 1.0 returns true
+    const magnitudeValue = parseFloat(magnitude);
+    const isMagnitudeNumeric = !Number.isNaN(magnitudeValue);
+    const isUnitSingular = !isMagnitudeNumeric || magnitudeValue === 1;
+
+    const measurement = `${magnitude} ${unit || ''}`.trim();
```

Comment:
Seems odd to me to add a space deliberately, then trim it if not needed. Couldn’t it be ${unit ? `${unit}` : ''}? I doubt there’s any real performance difference, but it just seems odd. I had one more outdated comment, but I see that one really doesn’t apply anymore.

Reply:
I added trim in the slim change there is a extra space before or after magnitude. The computational cost seemed low, so I tossed it in. I can take out trim if you insist. ${unit || '' and ${unit ? ` ${unit}` : ''} are equivalent. I selected the former because it is more brief. I can use what you have (${unit ? ` ${unit}` : ''}) if that is clearer.